content/soarian-financials/stu3/operation-definition.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
I am not going to look at this file at this time, apply all comments left on patient.md here